presto-main/src/main/java/com/facebook/presto/sql/planner/RelationPlanner.java

```diff
@@ -431,11 +431,7 @@ protected RelationPlan visitQuerySpecification(QuerySpecification node, Void con
     protected RelationPlan visitValues(Values node, Void context)
     {
         Scope scope = analysis.getScope(node);
-        ImmutableList.Builder<Symbol> outputSymbolsBuilder = ImmutableList.builder();
```

Comment:
In the future only `RelationType` (not the `Scope` itself) needs to be stored for `Node` in `Analysis`.